config/locales/en.yml

```diff
@@ -44,6 +44,8 @@ en:
         base: "Credit Card"
       order_cycle:
         orders_close_at: Close date
+      variant_override:
+        count_on_hand: "Count on Hand"
```

Comment:
I think it's better if the translation is "On hand" so that it is the same as the variant field.
So this would become:
`count_on_hand: "On Hand"`
Translators can still change it in transifex.

Reply:
change made